### PR TITLE
enhance trg 5-2

### PR DESCRIPTION
**Pull request by ricogu**
https://jira.catena-x.net/browse/A1ODT-1118

Add mandatory subsection 'Prerequisites' 
Should contain the kubernetes version, the helm version and every thing else which is not included in the helm chart

Add section similiar to TL;DR which contains at least the helm commands to add our helm chart repository (url from the helm chart repository)

### CODE REVIEW

File: docs/release/trg-5/trg-5-2.md [hunk 1]
```diff
@@ -104,6 +104,8 @@ The file **must** contain the [_Apache 2.0 Licence_](https://github.com/catenax-
 The README.md file **must** contain:
 
 - a brief explanation of what the Helm chart will install
+- a section of "Prerequisites" includes the required kubernetes version, helm version and other necessary prerequisites and their versions. See an example [here](https://github.com/bitnami/charts/tree/main/bitnami/postgresql#prerequisites).
+- a section of "TL;DR" that includes the helm commands to add anf install the helm chart, see example [here](https://github.com/bitnami/charts/tree/main/bitnami/postgresql#tldr)
```
Comment (carslen) [inline docs/release/trg-5/trg-5-2.md:108]: Not convinced that teams will know what to add (→ URL of Helm repo), also there is a typo.

**Review by carslen — CHANGES_REQUESTED** [review]

File: docs/release/trg-5/trg-5-2.md (hunk 1 above)
Comment (carslen) [reply docs/release/trg-5/trg-5-2.md]: In our TRG we should be more precise in what we expect. Examples in external repos might change without our knowledge.

Comment (ricogu) [reply docs/release/trg-5/trg-5-2.md]: changed to inline examples

**Review by carslen — APPROVED** [review]
LGTM

**Pull request merged**